src/unpackertool/plugins/upx/decompressors/decompressor.h

```diff
@@ -12,11 +12,9 @@
 
 #include "retdec/unpacker/decompression/compressed_data.h"
 
-namespace retdec {
+using namespace retdec::utils;
```

Comment:
Same as the comment I made earlier. Never put `using namespace` in header files.